src/components/search-bar/search-bar.tsx

```diff
@@ -24,6 +24,9 @@ const SearchBar: React.FC<SearchBarProps> = ({
       </label>
       {/* eslint-disable react/jsx-props-no-spreading */}
       <input
+        required
+        pattern=".*\S+.*"
```

Comment:
Is the pattern necessary?

Reply:
If it is not there, it is possible to make a search with whitespace characters.

Reply:
👍